site/en/guides/porting/define-the-platform-cmake-files.md

```diff
@@ -0,0 +1,13 @@
+# Define the platform CMake files
```

Comment:
I'm not sure how verbose this file should be

Reply:
We should keep it simple - the minimum to get a single platform up. I think we can leave multiple platform support out of scope.

Reply:
@lmnotran I agree with @jwhui - simple is better. Remove the multiple platform part then provide a few words for the single platform part and refer them to the appropriate efr32 info as an example.